sdk/core/core-rest-pipeline/CHANGELOG.md

```diff
@@ -1,6 +1,6 @@
 # Release History
 
-## 1.11.1 (Unreleased)
+## 1.11.1 (2023-08-01)
 
 ### Features Added
 
```

Comment:
you'll need to remove the empty sections (breaking changes and other changes) in order to release successfully. Running the [PrepareRelease](https://github.com/Azure/azure-sdk-for-js/blob/main/eng/common/scripts/Prepare-Release.ps1) script will do this for you.